file-share-service-desktop-client/Modules/Search/SearchViewModel.cs

```diff
@@ -105,9 +121,7 @@ private async Task ExecuteSearch()
                     };
                     batchDetailsVM.Add(bdvm);
                 }
-                RaisePropertyChanged(nameof(BatchDetailsVM));
-                NextPageCommand.RaiseCanExecuteChanged();
-                PreviousPageCommand.RaiseCanExecuteChanged();
+                logger.LogInformation("File Share Service search result Found :{Total} record.", SearchResult.Total);
             }
             catch (Exception e)
```

Comment:
Please log the caught exception with `LogError`

Reply:
resolved